.github/workflows/iOS-Sample.yml

```diff
@@ -0,0 +1,56 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build iOS Sample/Tutorial Swift and Objective-C apps
```

Comment:
It's a bit too long to see in the sidebar of the actions. Can you make this just "Build iOS apps"?